packages/web-components/components/accordion-group.ts

```diff
@@ -0,0 +1,8 @@
+import { defineCustomElement } from 'vue';
+import { PuikAccordionGroup } from '@prestashopcorp/puik-components';
+import type { CustomElementWithTagName } from '../types';
+
+const PuikAccordionGroupCe = defineCustomElement(PuikAccordionGroup) as CustomElementWithTagName;
+PuikAccordionGroupCe.tagName = 'puik-accordion-group-ce';
```

Comment:
I'm a bit confused about this, why not using the type returned by defineCustomElement and letting the user declare the tag name ? (This comment applies to all the other component files)

Reply:
Done by @mattgoud